builder/context/directive.go

```diff
@@ -35,10 +35,6 @@ func readDirectiveFile(cwd string) (*directive.Directive, error) {
 
 	_, err := os.Stat(filePath)
 	if err != nil {
-		if os.IsNotExist(err) {
```

Comment:
note:

I moved this out of the readDirectiveFile method to the build context because it felt out of place.

IMO if the caller has expressed the intent to read the directive but it can't be found this is an error; it is the caller's (build context) responsibility to decide what they want to do in this event.